tests/listsAndDropDowns.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Lists and Drop Downs', () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto('/')
+        await page.getByRole('button', { name: 'Owners' }).click()
+        await page.getByRole('link', { name: 'Search' }).click()
+        await expect(page.getByRole('button', { name: 'Owners' })).toHaveText('Owners');
+    })
+
+    test('Test Case 1: Validate selected pet types from the list', async ({ page }) => {
+        await page.getByRole('link', { name: 'George Franklin' }).click()
+        await expect(page.locator('.ownerFullName')).toHaveText('George Franklin')
+        await page.getByRole('button', { name: 'Edit Pet' }).click()
+        await expect(page.locator('#name')).toHaveValue('Leo')
+        await expect(page.locator('#owner_name')).toHaveValue('George Franklin')
+        await expect(page.locator('#type')).toHaveValue('cat')
+        const dropdownMenu = page.locator('#type')
+        await dropdownMenu.click()
+        const typesOfPets = ['cat', 'dog', 'lizard', 'snake', 'bird', 'hamster']
+
+        for (const pet of typesOfPets) {
+            await dropdownMenu.selectOption(pet)
+            await expect(dropdownMenu).toHaveValue(pet)
+            await expect(page.locator('#type')).toHaveValue(pet)
+            await dropdownMenu.click()
+        }
+    })
+
+    test('Test Case 2: Validate the pet type update', async ({ page }) => {
+        await page.getByRole('link', { name: 'Eduardo Rodriquez' }).click()
+        await page.getByRole('button', { name: 'Edit Pet' }).nth(1).click()
+        const typeOfPet = page.locator('#type')
+        const nameOfPet = page.locator('#name')
```

Comment:
Create a more descriptive naming for the constants that better reflects the reference. Later, it will help to easily read the code.
Instad of `typeOfPet` it would be `petTypeInputField` and instead of `nameOfPet` it would be `petNameInputField`